upcloud/client/client.go

```diff
@@ -2,216 +2,210 @@ package client
 
 import (
 	"bytes"
+	"context"
 	"crypto/tls"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
+	"strings"
 	"time"
 
-	globals "github.com/UpCloudLtd/upcloud-go-api/v4/internal"
-	"github.com/blang/semver"
 	"github.com/hashicorp/go-cleanhttp"
 )
 
-// Constants
 const (
-	DefaultAPIVersion = "1.3.6"
-	DefaultAPIBaseURL = "https://api.upcloud.com"
-
-	// The default timeout (in seconds)
-	DefaultTimeout = 60
+	Version    string = "5.0.0"
```

Comment:
This replaces the `internal/globals.go` right? If so, we also need to update RELEASING.md to point to this line in step 3.

Reply:
@ka-myl Yep, I thought that there is no reason to keep separate package for this.  I'll update RELEASING.md if this is otherwise OK ?